index.js

```diff
@@ -8,7 +8,9 @@ const PORT = process.env.PORT || 3000
 app.set("view engine", "ejs")
 
  
-
+app.get("/", (req, res)=>{
```

Comment:
What is the use of this line of code?

Reply:
I think you forgot that you suggested to do that